rust/fory-core/src/types.rs

```diff
@@ -95,8 +95,10 @@ pub enum TypeId {
     U64_ARRAY = 75,
     USIZE_ARRAY = 76,
     U128_ARRAY = 77,
+    INT128 = 78,
+    ISIZE = 79,
```

Comment:
do we still need this type id? and coudk we also remvoe USIZE type id?

Reply:
I already add these back, let me add some docs

Reply:
Done